k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -259,16 +253,42 @@ buildValues() {
         fi
     done
 
-    for c in data client; do
-        if [[ ${components["${c}"]} == true ]]; then
-            for node in "${node_name_list[@]}"; do
-                uuid=$(< "${cfgmap_path}/auto-gen-${node}-${namespace}/${c}/id")
-                yq -i eval-all "
-                    select(fi==0) ref \$to | select(fi==1).solution.common.storage_sets.name as \$name
-                    | \$to.configmap.clusterStorageSets.[\$name].nodes.${node}.${c}Uuid=\"${uuid}\"
-                    | \$to" "${values_file}" "${solution_yaml}"
-            done
-        fi
+    if [[ ${components["client"]} == true ]]; then
+        for node in "${node_name_list[@]}"; do
+            uuid=$(< "${cfgmap_path}/auto-gen-${node}-${namespace}/client/id")
+            yq -i eval-all "
+                select(fi==0) ref \$to | select(fi==1).solution.common.storage_sets.name as \$name
+                | \$to.configmap.clusterStorageSets.[\$name].nodes.${node}.Uuid=\"${uuid}\"
```

Comment:
```suggestion
                | \$to.configmap.clusterStorageSets.[\$name].nodes.${node}.clientUuid=\"${uuid}\"
```
[_cluster.yaml.tpl](https://github.com/Seagate/cortx-k8s/pull/286/files#diff-96c4b2b43d4bef978c12020daf756740783392143a62bf8f194cc92c0bc5a409) is still using `clientUuid`.

Reply:
Thanks for the catch here. I explicitly updated that section due to another Codacy comment, but it must have just wiped out the "client" string altogether.